File: p020/p020.py
def sumFactorialDigits(n):
    bigNum = [1]
    for i in range(2, n + 1):
        count = len(bigNum)
        overflow = 0
        for j in range(count):
            digit  = bigNum[j]
            digit = digit * i + overflow
            overflow = digit // 10
            bigNum[j] = digit % 10
        while overflow > 0:
            bigNum.append(overflow % 10)
            overflow = overflow // 10
    return sum(bigNum)

if __name__ == "__main__":
    print(sumFactorialDigits(100))

File: p020/p020.js
function sumFactorialDigits(n) {
  // Good luck!
  const bigNum = [1];
  for (let i = 2; i <= n; i++) {
    let count = bigNum.length;
    let overflow = 0;
    for (let j = 0; j < count; j++) {
      let digit = bigNum[j];
      digit = digit * i + overflow;
      overflow = Math.floor(digit / 10);
      bigNum[j] = digit % 10;
    }
    while (overflow > 0) {
      bigNum.push(overflow % 10);
      overflow = Math.floor(overflow / 10);
    }
  }
  return bigNum.reduce((acc, v) => acc + v);
}

console.log(sumFactorialDigits(100));
